packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -487,24 +512,30 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 		openChat();
 
 		streaming.value = true;
+		const payload: ChatRequest.RequestPayload['payload'] = {
+			role: 'user',
+			type: 'init-error-helper',
+			user: {
+				firstName: usersStore.currentUser?.firstName ?? '',
+			},
+			error: context.error,
+			node: assistantHelpers.processNodeForAssistant(context.node, [
+				'position',
+				'parameters.notice',
+			]),
+			nodeInputData,
+			executionSchema: schemas,
+			authType,
+		};
+		const payloadSize = getObjectSizeInKB(payload);
+		if (payloadSize > AI_ASSISTANT_MAX_CONTENT_LENGTH) {
```

Comment:
same question here. If we remove parameters, but object size is still too big, should not we check again until size is small enough and remove more things or fallback with an error?